MonoGame.Framework/Audio/NoMicrophoneConnectedException.cs

```diff
@@ -0,0 +1,36 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.Serialization;
+
+namespace Microsoft.Xna.Framework.Audio
+{
+
+    /// <summary>
+    /// The exception thrown when no audio hardware is present, or driver issues are detected.
+    /// </summary>
+    [DataContract]
+#if WINRT
+    public sealed class NoMicrophoneConnectedException : Exception
+#else
+    public sealed class NoMicrophoneConnectedException : ExternalException
```

Comment:
Wait, don't change it, turns out `ExternalException` does exist in .Net core, but it's currently not in the stable branch (it will be included in .Net Core 2.0).